README.md

```diff
@@ -60,9 +60,9 @@ For this plugin to operate you must install at least one of the dependent librar
 
 **You must install the dependencies for at least one of the supported secrets providers or a `RuntimeError` will be raised.**
 
-#### AWS Secrets Manager
+#### AWS Secrets Manager, AWS System Manager Parameter Store
```

Comment:
Having both names in the same heading is a bit awkward to me. Can you please change this to just "AWS" and then add  a sentence here to the effect of:


"AWS Secrets Manager and System Manager Parameter Store are supported."

Reply:
Also, can you please add a row for "AWS System Manager Parameter Store" to the table under the "Supported Secrets Backends" heading at the top of this file?

Reply:
I agree with this suggestion. Also it looks like `System Manager` should be `Systems Manager` throughout?